plugins/kubernetes-backend/api-report.md

```diff
@@ -31,6 +31,7 @@ export interface ClusterDetails {
   // (undocumented)
   caData?: string | undefined;
   dashboardApp?: string;
+  dashboardParameters?: any;
```

Comment:
I think for example `Record<string, string>` could be a good start here instead, to keep thins a bit more strict. Another option is `JsonObject` from `@backstage/types`

Reply:
ok, will do

Reply:
Done. Thanks for the suggestion since I discovered a bug when reading options from the Config! I've added the missing unit tests to avoid future regressions.